### PR TITLE
Use Stream over LightDB Stream

### PR DESCRIPTION
**Pull request by szczys**
* Rename `examples/zephyr/lightdb_stream` sample to `examples/zephyr/stream`
* Treewide: change all LightDB Stream references to Stream

This is ready for review but I've added the DNM tag so that it stays open until we're ready to merge as part of a larger release.

resolves https://github.com/golioth/firmware-issue-tracker/issues/524

### CODE REVIEW

File: examples/modus_toolbox/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This example supports all Golioth services, including Over-the-Air firmware
 updates:
 
 * LightDB: https://docs.golioth.io/cloud/services/lightdb
-* LightDB Stream: https://docs.golioth.io/cloud/services/lightdb-stream
+* Stream: https://docs.golioth.io/cloud/services/stream
```
Comment (hasheddan) [inline examples/modus_toolbox/README.md:25]: Let's go ahead and make this update as the page already exists and will be what we want to be pointing to shortly.
```suggestion
* Stream: https://docs.golioth.io/data-routing
```

File: examples/zephyr/stream/README.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
-# Golioth LightDB Stream sample
+# Golioth Stream service sample
```
Comment (hasheddan) [inline examples/zephyr/stream/README.md:1]: nitpick: I think leaving out "service" is fine here
```suggestion
# Golioth Stream sample
```

File: examples/zephyr/stream/README.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
-do not have temperature sensor a value is generated from 20 up to 30.
+This Golioth Stream application demonstrates how to connect with Golioth
+and periodically send data to the Stream service. In this sample
+temperature measurements will be displayed on the `/temp` Stream
+path. For platforms that do not have a temperature sensor, a value is
+generated from 20 up to 30.
+
+> :information-source: To receive Stream data, a Pipeline must be
+> configured using the Golioth Web Console. This example is configured
+> to work with the default Pipeline.
```
Comment (hasheddan) [inline examples/zephyr/stream/README.md:13]: Was thinking about potentially removing the part about the default pipeline here and just putting the YAML for the pipeline directly in this README. That would be useful if you had previously deleted the default, or the default changes at some point in the future.
```suggestion
> :information-source: To receive Stream data, a Pipeline must be
> configured for the project on Golioth. This example is configured
> to work with the default Pipeline present for every project upon creation.
```

File: examples/zephyr/stream/README.md [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
+# Golioth Stream service sample
 
 ## Overview
 
-This LightDB Stream application demonstrates how to connect with Golioth
-and periodically send data to LightDB Stream. In this sample temperature
-measurements are sent to `/temp` LightDB Stream path. For platforms that
-do not have temperature sensor a value is generated from 20 up to 30.
+This Golioth Stream application demonstrates how to connect with Golioth
+and periodically send data to the Stream service. In this sample
```
Comment (sam-golioth) [inline examples/zephyr/stream/README.md:6]: Are we using "Stream" as a product name? Or just to describe a kind of data (i.e. "stream data")?

Comment (hasheddan) [reply examples/zephyr/stream/README.md:6]: @sam-golioth just to describe a kind of data, so I think saying `Golioth data streaming application` here may be a bit easier to understand 👍🏻

**Review by mniestroj — APPROVED** [review]

**Review by sam-golioth — APPROVED** [review]

**Pull request merged**